src/metabase/driver/crate/native.clj

```diff
@@ -0,0 +1,31 @@
+(ns metabase.driver.crate.native
+  (:require [clojure.java.jdbc :as jdbc]
+            [metabase.models.database :refer [Database]]
+            [metabase.db :refer [sel]]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.tools.logging :as log]
```

Comment:
do you mind putting the `clojure` `:requires` above the Metabase ones? We keep them ordered like:

1.  Clojure namespaces
2.  3rd-party namespaces
3.  Metabase namespaces